content/integrations/docker.md

```diff
@@ -19,7 +19,7 @@ Get metrics from Docker in real time to:
 
 # Installation
 
-There are three ways to setup the Docker integration: install the agent on the host, on a single privileged container, and on each individual container.
+There are two ways to setup the Docker integration: install the agent on the host or on a single sidecar container.
```

Comment:
The goal of this PR is specifically remove that way from official documentation, as this creates more issues than it solves, and several clients were mislead by that.
Indeed, some integrations need work, but we should keep that container-as-new-host method as a KB and not a public documentation.